client/app/services/session.service.js

```diff
@@ -87,7 +94,7 @@
         services:    {show: entitledForServices(productFeatures)},
         requests:    {show: entitledForRequests(productFeatures)},
         marketplace: {show: entitledForServiceCatalogs(productFeatures)},
-        blueprints:  {show: false}
+        blueprints:  {show: entitledForServiceCatalogs(productFeatures) && $state["blueprints_flag"]}
```

Comment:
@h-kataria this should be checking the feature that give the Catalog Items accordion in the OPS UI